circleguard/widgets.py

```diff
@@ -765,3 +771,24 @@ def __init__(self, text):
         layout.addWidget(QLabel(text), 0, 1, 1, 1)
 
         self.setLayout(layout)
+
+
+class EntryWidget(QFrame):
+    button_pressed = pyqtSignal(object)
+    """
+    Represents a single entry, consisting of a title, an action name and data which is stored for ease of use.
```

Comment:
super vague...an entry for what? data for what? what does "action name" even mean?